tests/flows/fulfillment/test_reseller_change.py

```
@@ -184,6 +187,109 @@ def test_check_adobe_reseller_transfer_step_pending_status(
     mock_next_step.assert_not_called()
 
 
+def test_check_adobe_reseller_transfer_step_no_line_items_fallback_to_purchase(
+    mocker,
```

Ruff (0.15.1)

[warning] Unused function argument: `mocker`

(ARG001)